static/js/ducks/sources.js

```diff
@@ -73,6 +101,10 @@ messageHandler.add((actionType, payload, dispatch, getState) => {
       dispatch(fetchFavoriteSources());
     }
   }
+  const { gcnEvent } = getState();
```

Comment:
What if there isn't a gcn event loaded? Won't this still trigger all gcn events being fetched?

Reply:
What do you suggest here?

Reply:
Only do the dispatch below if `gcnEvent` is not null/empty/etc.

Reply:
Ok I think that hopefully fixed it.

Reply:
Resolving in favor of comment below